mail_tracking/static/src/js/discuss/discuss.js

```diff
@@ -28,6 +27,7 @@ odoo.define("mail_tracking/static/src/js/discuss/discuss.js", function (require)
                         ],
                     ],
                 });
+                await this.async(() => this._super(...arguments));
```

Comment:
Since you do not perform asynchronous operations, it should be sufficient to continue the chain of promises normally, right?
```suggestion
                return this._super(...arguments);
```

Reply:
no, because the super is not returning a promise

![image](https://user-images.githubusercontent.com/35231827/149962220-e3f7a866-edb1-4668-9684-ab723294656a.png)

Reply:
right, done.